invenio_communities/members/services/schemas.py

```diff
@@ -141,3 +151,28 @@ def get_permissions(self, obj):
             # owners/managers, prior value of visibility, self, prior
             'can_update_visible': self.is_self(obj),
         }
+
+
+class InvitationDumpSchema(MemberDumpSchema):
+    """Schema for dumping invitations."""
+
+    # request = fields.Nested(RequestSchema)
+    request = fields.Method('get_mocked_request')
+    permissions = fields.Method('get_permissions')
+
+    def get_mocked_request(self, obj):
+        # Temporary method for mocking the results.
+        return {
+            "id": obj.request_id,
+            "status": "submitted",
+            "expires_at": "2023-04-08T15:24:05.688992+00:00",
+        }
+
+    def get_permissions(self, obj):
+        """Get permission"""
+        # TODO: should take request status into account - if you can view an
```

Comment:
For this one, we first need to index the status field of the request into the index (similar to users/groups).